samples/aspnetcore-authz-with-ext-authz-service/Contoso.API/Program.cs

```diff
@@ -0,0 +1,58 @@
+using Contoso.API.Authorization;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.Identity.Web;
+using Microsoft.Identity.Web.Resource;
+
+namespace Contoso.API
+{
+    public class Program
```

Comment:
Use template generated Program.cs

Reply:
> @Rick-Anderson - This is template generated. Can you explain?

Create an app with the templates, that is with Visual Studio or `dotnet new`. We should always start with the tool generated code.

Reply:
Yes, this solution/project is generated with Visual Studio. I went ahead and tested with dotnet new, and it seemed dotnet new produces a version with minimal api.